spec/serializers/ingredients/index_serializer_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe Ingredient::IndexSerializer, type: :serializer do
+  let(:ingredients) { create_list(:ingredient, 20) }
```

Comment:
Here, on the other hand, there's no need to create 20 objects. You are not using them for anything. I would argue that just 1 is enough